commands.js

```diff
@@ -194,6 +190,12 @@ let takeScreenshot = (element, name, modifiedOptions, win) => {
                 }
             }
 
+
+            [numScrolls, offsetHeight, viewportHeight, viewportWidth] = getImageDimension(win)
```

Comment:
move this to the actual lazyload block, this currently would be ran for all fullpage screenshots which is not needed